src/App.jsx

```diff
@@ -2,14 +2,16 @@ import { Route, Routes } from "react-router-dom";
 
 import { Footer, Navbar } from "./Components";
 
-import { Home } from "./Pages";
+import { Home, Courses, Error } from "./Pages";
 
 const App = () => {
   return (
     <>
       <Navbar />
       <Routes>
-        <Route path="/" element={<Home />} />
+        <Route path="*" element={<Error />} />
```

Comment:
Error component should be the last route. Move it to the bottom.

Reply:
Fixed